app.js

```diff
@@ -1,3 +1,5 @@
+/* Hometask1 */
+
 let usersList = [];
 
 const MyTable = ({ MyName, MySurname, MyLastname}) => {

```

Comment:
use `camelCase` for object keys please (first letter is not capitalized)

Reply:
Done